modules/client/test/unit/context.test.ts

```diff
@@ -27,54 +36,119 @@ describe("Context instances", () => {
       web3Providers: web3endpoints.working,
       gasFeeEstimationFactor: 0.1,
       graphqlNodes: [],
+      ipfsNodes: [],
     };
   });
-  it("Should create an empty context", () => {
-    const context = new Context({});
-
-    expect(context).toBeInstanceOf(Context);
-  });
   it("Should create an empty context and have default values", () => {
-    const context = new Context({});
-
+    const context = new Context();
     expect(context).toBeInstanceOf(Context);
-    expect(context.signer).toEqual(undefined);
-    expect(context.gasFeeEstimationFactor).toEqual(0.625);
+    expect(context.network.name).toBe("homestead");
+    expect(context.network.chainId).toBe(1);
+    expect(context.daoFactoryAddress).toBe(LIVE_CONTRACTS.homestead.daoFactory);
+    expect(context.ensRegistryAddress).toBe(context.network.ensAddress);
+    expect(context.gasFeeEstimationFactor).toBe(0.625);
+    expect(context.web3Providers.length).toBe(WEB3_NODES.homestead.length);
+    expect(context.ipfs.length).toBe(IPFS_NODES.homestead.length);
+    expect(context.graphql.length).toBe(GRAPHQL_NODES.homestead.length);
+    context.web3Providers.map((provider) => {
+      expect(provider).toBeInstanceOf(JsonRpcProvider);
+    });
+    context.ipfs.map((ipfsClient) => {
+      expect(ipfsClient).toBeInstanceOf(IpfsClient);
+    });
+    context.graphql.map((graphqlClient) =>
+      expect(graphqlClient).toBeInstanceOf(GraphQLClient)
+    );
   });
   it("Should create a context and have the correct values", () => {
     const context = new Context(contextParams);
 
     expect(context).toBeInstanceOf(Context);
-    expect(context.network).toEqual("mainnet");
-    expect(context.signer).toBeInstanceOf(Wallet);
-    expect(context.daoFactoryAddress).toEqual("0x1234");
-    context.web3Providers?.map((provider) =>
+    expect(context.network.name).toBe("homestead");
```

Comment:
Why don't we try with values that are actually different to the default ones?

Reply:
I'mean they are different this is using the contextParams defined at the top of the file, in the next test we test with another parameters

Reply:
Right, but what I like about tests is the fact that you can do a clear A/B test within the same test case. If you have to look 3 different scopes to realize that this is covered, that's hard to follow and make sense of

Reply:
Im not sure I'm following what you mean. We have 3 tests to test values:

- First we pass an empty value and check that sets the deaults
- Second que pass specific contextParams and check that the context sets them properly
- Third we have a context with some parameters and we set new params and check that they were set properly

You want to group this tests in one?